scripts/pagespeed_libraries_generator.sh

```diff
@@ -27,9 +27,9 @@
 # Author: vid@zippykid.com (Vid Luther)
 #         jefftk@google.com (Jeff Kaufman)
 
-URL="https://github.com/pagespeed/mod_pagespeed/raw/master/"
-URL+="net/instaweb/genfiles/conf/pagespeed_libraries.conf"
-curl -L -s -S "$URL" \
+URL="https://raw.githubusercontent.com/pagespeed/mod_pagespeed/master/"
+URL2="net/instaweb/genfiles/conf/pagespeed_libraries.conf"
```

Comment:
Sorry -- introducing `URL2` here is not needed, you can reuse URL (see the original code).
And then below you can go 
```
curl -L -s -S "$URL" \
```

instead of 

```
curl -L -s -S "$URL""$URL2" \
```